### ISSUE TITLE
Issue #72, part 3

### ISSUE THREAD

**Issue by boaks** [OpenMobileAlliance/OMA_LwM2M_for_Developers#111]
```
Subject : OMA-TS-leightweightM2M-V1_0-20141126C

3.) the example below this paragraph - should this not read
 </1/101>, </1/102>, </2/0>, </2/1>, </2/2>, </2/3>, </2/4>, </3/0>, </4/0>, </5>?
 (The reasoning for the </5> is not explained anywhere
```

```
3.) The line on page 20 referring to the example in Appendix F reads
 </1/101>", </1/102>, </2/0>, </2/1>, </2/2>, </3/0>, </4/0>, </5>
The instantiated objects in the example in Appendix F are actually
 </1/101>, </1/102>, </2/0>, </2/1>, </2/2>, </2/3>, </2/4>, </3/0>, </4/0>. In other words there are some instantiated objects missing on page 20 (at least according to my understanding). 
```

```
OMA-TS-LightweightM2M-V1_0-20160407-C, 5.3.1, page 27
</1/0>, </1/1>, </2/0>, </2/1>, </2/2>, </3/0>, </4/0>, </5>
...
</lwm2m>;rt="oma.lwm2m", < /1/101>, < /1/102>, < /2/0>, < /2/1>, < /2/2>, < /3/0>,< /4/0>,< /5> 

```
still doesn't cover the example in appendix F!
Appendix F, page 116, table 27:
<0/0>, <0/1>, <0/2>, <1/1>, <1/2>, <2/0>, <2/1>, <2/2>, <2/3>, <2/4>, <3/->, <4>

Note1: #93 should clarify, if the <0/?> should be reported.
Note2: the instances of LWM2M Server <1/?> according the example are "1" and "2", not to be mixed up with the short server id ("101" and "102").
Note3: don't know, why the device object instance id is "-", I would say, "0".

**Comment by ThGarnier** [comment]
These points are now covered in Oct16 TS => to be closed

**Comment by boaks** [comment]
Can you provide a note, why firmware update (/5) is supported, but no instance (/5/0) is created?
Use case?

**Comment by dnav** [comment]
The Client declares it supports Firmware Update mechanism but the associated hardware and software resources are not reserved for it yet.
When the Server creates the instance /5/0, the Client can now reserve the needed HW and SW resources like loading required sharing libraries and preparing storage space.

**Comment by boaks** [comment]
Thanks!
So fixed with OMA-TS-LightweightM2M-V1_0-20161007-D.
